src/libostree/ostree-repo.c

```diff
@@ -4975,8 +4975,19 @@ repo_list_objects_impl (OstreeRepo                  *self,
   return g_steal_pointer (&ret_objects);
 }
 
-/* A currently-internal version of ostree_repo_list_objects which returns
- * a set, and not a map (with a useless value).
+/* ostree_repo_list_objects_set:
+ * @self: Repo
+ * @flags: Flags controlling enumeration
+ * @cancellable: Cancellable
+ * @error: Error
+ *
+ * This function is similar to ostree_repo_list_objects but returns a
+ * set and not a map (with a useless value).
+
+ * Returns: (transfer container) (element-type GVariant):
```

Comment:
I did `transfer container` here to match `ostree_repo_list_objects` but I think both of them should probably be `transfer full` instead. But I guess it doesn't matter for `GHashTable` since the container is what owns the objects inside anyway.